docs/react-v9/contributing/rfcs/react-components/convergence/extended-design-tokens.md

```diff
@@ -0,0 +1,442 @@
+| **Option A: Update existing components with var() fallbacks** | • Existing consumers automatically get the FST system without additional work beyond updating their packages<br>• Everyone stays in sync and should have equal capability improving interop and portability | • Once the changes are made, we are somewhat stuck with them (until the next major version)<br>• All consumers will pull these changes which means some increases in bundle size (gzipped impact seems minor based on our test PR so far).                                                                                                                                                                                                                                                               |
+| Option B: Custom style hooks                                  | • Fully opt in<br>• Removes component changes at the root and could allow us to ship separately from the core packages meaning breaking changes could be managed more easily in a separate package          | • performance regressions were noted in testing. This might be due to the fact that styles were generated and inserted in runtime. Additionally, it depends on React's context which re-renders components.<br>• Because it's opt in we wouldn't have uniform adoption<br><br>Opportunity: If we can fix the performance issues or understand them better this might be a really interesting option with lower risk. Can we investigate using these styles hooks but with Griffel outputting static CSS? |
+| Option C: Create component variants through recomposition     | • Could be shipped in separate packages<br>• avoids breaking changes                                                                                                                                        | • Could create confusion (which Button do consumers use? The base one or our SemanticTokenButton?)<br> There could be confusing ways in which semantic tokens could work in some places but not others if the right variants aren't used.                                                                                                                                                                                                                                                                |
+
+## Open questions
+
+In-product performance testing
+
+To date, we’ve mostly conducted synthetic tests against var() fallback performance. We still need to run integration tests against product scenarios to test real-world effects of additional CSS variables and fallbacks, as well as the overall dev experience. We plan to test this in Teams and will update accordingly.
+
+### Package location
+
+Where should the expanded tokens live? We’re leaning towards Option B (**bolded**) now but would like feedback.
```

Comment:
Option A is bolded 😉

Reply:
You aren't wrong :D. Yah I think we originally had B but after going back and forth and syncing with Levi we went to A. Thanks for the catch!